lib/tasks/populate_db.rake

```diff
@@ -0,0 +1,68 @@
+require 'factory_bot'
```

Comment:
this is fantastic, we had a ticket for this FYI

https://github.com/ophrescue/RescueRails/issues/646

Reply:
FactoryGirl always troubled me, I'm glad they changed the name.